splice_si_api/src/main/java/com/splicemachine/si/impl/TransactionImpl.java

```diff
@@ -60,58 +60,20 @@ public int setSavePoint(String name,Object kindOfSavepoint) throws IOException{
             ignoreSavePoints=true;
         }
 
-        if(LOG.isDebugEnabled())
-            SpliceLogUtils.debug(LOG,"Before setSavePoint: name=%s, savePointStack=\n%s",name,getSavePointStackString());
+//        if(LOG.isDebugEnabled())
+//            SpliceLogUtils.debug(LOG,"Before setSavePoint: name=%s, savePointStack=\n%s",name,getSavePointStackString());
         setActiveState(false,false,null); //make sure that we are active
         Txn currentTxn=getTxn();
         Txn child=lifecycleManager.beginChildTransaction(currentTxn,null);
         txnStack.push(Pair.newPair(name,child));
-        if(LOG.isDebugEnabled())
-            SpliceLogUtils.debug(LOG,"After setSavePoint: name=%s, savePointStack=\n%s",name,getSavePointStackString());
+//        if(LOG.isDebugEnabled())
+//            SpliceLogUtils.debug(LOG,"After setSavePoint: name=%s, savePointStack=\n%s",name,getSavePointStackString());
```

Comment:
I removed them, now we end up with huge stacks that take a long time to serialize into a String.